### PR TITLE
Reference Unsloth in header

### PR DESCRIPTION
**Pull request by momochen**
## Summary
Reference Unsloth in header section

## Testing Done

- Hardware Type: <BLANK>
- [x] run `make test` to ensure correctness
- [x] run `make checkstyle` to ensure code style
- [x] run `make test-convergence` to ensure convergence

### CODE REVIEW

**Comment by momochen** [comment]
@ByronHsu @helloworld1 @yundai424 @qingquansong ptal

**Review by qingquansong — APPROVED** [review]

**Review by lancerts — APPROVED** [review]

**Comment by helloworld1** [comment]
In apache license, it says ```If the Work includes a "NOTICE" text file as part of its distribution, then any Derivative Works that You distribute must include a readable copy of the attribution notices contained within such NOTICE file, excluding those notices that do not pertain to any part of the Derivative Works, in at least one of the following places: within a NOTICE text file distributed as part of the Derivative Works; within the Source form or documentation, if provided along with the Derivative Works; or, within a display generated by the Derivative Works, if and wherever such third-party notices normally appear. The contents of the NOTICE file are for informational purposes only and do not modify the License. You may add Your own attribution notices within Derivative Works that You distribute, alongside or as an addendum to the NOTICE text from the Work, provided that such additional attribution notices cannot be construed as modifying the License. ``` Are those two files considered derived work? If so, we also need to change the NOTICE file.

File: src/liger_kernel/ops/rms_norm.py
Comment (danielhanchen) [inline src/liger_kernel/ops/rms_norm.py:1]: This looks reasonable for referencing Unsloth code.

File: src/liger_kernel/ops/utils.py
Comment (danielhanchen) [inline src/liger_kernel/ops/utils.py:1]: This looks reasonable for referencing Unsloth code.

**Comment by momochen** [comment]
> In apache license, it says
> 
> If the Work includes a "NOTICE" text file as part of its distribution, then any Derivative Works that You distribute must include a readable copy of the attribution notices contained within such NOTICE file, excluding those notices that do not pertain to any part of the Derivative Works, in at least one of the following places: within a NOTICE text file distributed as part of the Derivative Works; within the Source form or documentation, if provided along with the Derivative Works; or, within a display generated by the Derivative Works, if and wherever such third-party notices normally appear. The contents of the NOTICE file are for informational purposes only and do not modify the License. You may add Your own attribution notices within Derivative Works that You distribute, alongside or as an addendum to the NOTICE text from the Work, provided that such additional attribution notices cannot be construed as modifying the License.
> 
> Are those two files considered derived work? If so, we also need to change the NOTICE file.

Sure will followup with another ticket to change NOTICE file accordingly.

**Comment by momochen** [comment]
We are glad to announce that Unsloth and LinkedIn have fully resolved disagreements regarding references!

**Pull request merged**